conan/tools/build/flags.py

```diff
@@ -1,11 +1,26 @@
+from conan.errors import ConanException
 from conan.internal.model.version import Version
 
 
+def disable_flag(conanfile, flag):
+    disable_flags = conanfile.conf.get("tools.gnu:disable_flags", check_type=list)
+    if disable_flags is None:
+        return False
```

Comment:
I'd simply allow something like:

```suggestion
        return False
    elif disable_flags == ['*']:
        return True
```

Reply:
But we agreed to not add anything yet to implement the "all" concept, but wait for users feedback and request. Do we want to reconsider?

Reply:
After reviewing it, I thought that it could be fine. Anyway, let's wait for it as we agreed, ok 👍